go-controller/hack/build-go.sh

```diff
@@ -23,7 +23,10 @@ build_binaries() {
     set -x
     for bin in "$@"; do
         binbase=$(basename ${bin})
-        go build -v \
+	if [[ "$binbase" == "ovn-k8s-cni-overlay" ]]; then
+	    CGO_ENABLED=0
+	fi
```

Comment:
does this mean that in the subsequent iteration of the loop, the CGO_ENABLED will still be set to 0? in other words, it could be that CGO_ENABLED=0 for other binaries too if they are build after `ovn-k8s-cni-overlay`?

Reply:
yeah that is not right I nneed to put back to the default value Thanks!!

Reply:
I'd just enable it by default and disable it based on the condition, no real need for the if else.

Reply:
GCO is enabled by default but once we disable it when build for ovn cni we need to set it back for the result of the modules in the for loop ? do suggest 
for .....; do
.....
CGO_ENABLED=1
if [[ ...]]; then
CGO_ENABLED=0
fi

Reply:
Yep, something like that would work, no need to specifically go case by case. Thanks!